manifests/install.pp

```diff
@@ -9,6 +9,9 @@
 
   file { $fluentd::config_path:
     ensure  => directory,
+    owner   => 'root',
+    group   => 'root',
+    mode    => '0755',
```

Comment:
Configuration file should not be world readable. 0750 is IMHO better mode ... or implement possibility to override the default mode.